src/gui/newwizard/setupwizardwindow.ui

```diff
@@ -101,7 +101,7 @@
         <item>
          <widget class="QPushButton" name="errorMessageDismissButton">
           <property name="text">
-           <string>Dismiss</string>
+           <string>&amp;Dismiss</string>
```

Comment:
Apparently Mac doesn't like accessibility, see `Continue, `Back` and `Finish` ....
So I'm a bit lost here... 🙈

Reply:
Mac ignores those, they only show up on Windows and Linux.